stratum/lib/security/auth_policy_checker.cc

```diff
@@ -6,9 +6,17 @@
 
 #include <errno.h>
 #include <stdlib.h>
+
+// The current (stub) implementation of AuthPolicyChecker uses Linux specific
+// syscalls. To allow building on other platforms, we provide noop stubs for the
+// parts that depend on Linux.
+// TODO(max): The file watcher functions don't seem to be used anywhere.
```

Comment:
I'm guessing that these were originally added for the `Authorize()` and `RefreshPolicies()` methods below. 

I'm okay just removing them entirely, then when someone wants to implement the files below we have them introduce something more platform independent. For example, https://github.com/SpartanJ/efsw

Reply:
Seems reasonable to me as well.